### PR TITLE
Invert gain

### PR DESCRIPTION
**Pull request by davidthomas5412**
Inverted the gain to be consistent with Phosim. See page 13 of https://arxiv.org/pdf/1504.06570.pdf for the formula PhoSim uses. Note that for our simulations N, the nonlinear contribution, is set to 0.

Also updated flake8 ignores based on LSST guidelines.

### CODE REVIEW

File: setup.cfg [hunk 1]
```diff
@@ -1,6 +1,6 @@
 [flake8]
 max-line-length = 110
-ignore = E133, E226, E228, N802, N803, N806
+ignore = E133, E226, E228, N802, N803, N806, N812, N813, N815, N816, W504
```
Comment (SimonKrughoff) [inline setup.cfg:3]: Out of curiosity, why do we need the extra exceptions?

**Review by SimonKrughoff — APPROVED** [review]
Looks good.

File: setup.cfg (hunk 1 above)
Comment (timj) [reply setup.cfg:3]: Because I updated pep8-naming in February. I fixed all the lsst_distrib packages but I didn't fix ones I didn't know about.

Comment (SimonKrughoff) [reply setup.cfg:3]: Ah.  OK.  Thanks for the explanation.

**Pull request merged**